packages/app/src/cli/services/dev/extension/utilities.ts

```diff
@@ -33,6 +33,11 @@ export function getExtensionPointTargetSurface(extensionPointTarget: string) {
         return 'post_purchase'
       }
 
+      // The Thank You page is rendered as part of the checkout application
+      if (page === 'thank-you') {
+        return 'checkout'
+      }
```

Comment:
Should the default be `checkout` for all purchase extension points and we can add exceptions if that's not the case ?

Just checking quickly, we're going to have the same issue with: `purchase.cart-line-item.line-components`

Reply:
Good call, will update